doc/articles/features/pointers-keyboard-and-other-user-inputs.md

```diff
@@ -0,0 +1,183 @@
+# Uno support for user inputs
+
+## Supported user inputs
+
+User inputs are usually propagated to the using `RoutedEvent`, see. Uno's [routed events documentation](routed-events.md) to better understand their implementation on Uno.
+
+| Routed Event                  | Android | iOS     | Wasm    | MacOS | Skia WPF | Skia GtK | Tizen |    |
```

Comment:
I think there is a mismatch between the table header and column count. The table doesn't render correctly to HTML.